<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>750</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>750</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>750</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Termina partite</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #FCD19C</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>781</width>
     <height>721</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>150</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>150</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../../Logo_bowling.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="terminaButton">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>28</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(135, 75, 15);</string>
          </property>
          <property name="text">
           <string>Termina partita</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListWidget" name="partiteList">
        <property name="styleSheet">
         <string notr="true">color: black</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="indietroButton_2">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(135, 75, 15);</string>
        </property>
        <property name="text">
         <string>&lt; indietro</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
